tests/test_postgresql.py

```diff
@@ -133,6 +133,11 @@ def test_create_index_if_not_exists(self):
         op.create_index("i", "t", ["c1", "c2"], if_not_exists=True)
         context.assert_("CREATE INDEX IF NOT EXISTS i ON t (c1, c2)")
 
+    def test_create_fk_postgresql_not_valid(self):
+        context = op_fixture("postgresql")
+        op.create_foreign_key("i", "t1", "t2", ["c1"], ["c2"], postgresql_not_valid=True)
```

Comment:
this is not a render test, this is just an op test, which is fine, but this test should be passing already.   you'd need to have tests in tests/test_autogen_render.py , see test_render_drop_index_custom_kwarg for an example of how to test this